repos/fdbt-site/src/pages/products/exports.tsx

```diff
@@ -52,21 +53,25 @@ const getTag = (exportDetails: Export): ReactElement => {
     );
 };
 
-const Exports = ({ csrf, operatorHasProducts }: GlobalSettingsProps): ReactElement => {
+const Exports = ({ csrf, exportStarted, operatorHasProducts }: GlobalSettingsProps): ReactElement => {
     const [showExportPopup, setShowExportPopup] = useState(false);
     const [showFailedFilesPopup, setShowFailedFilesPopup] = useState(false);
     const [buttonClicked, setButtonClicked] = useState(false);
+    const [isExportStarted, setIsExportStarted] = useState(exportStarted);
```

Comment:
instead of isExportStarted, lets call it 'hasExportStarted' and 'setHasExportStarted'